frontend/public/module/k8s/types.ts

```diff
@@ -810,7 +810,7 @@ export type Release = {
 export type VersionUpdate = { version?: string; image?: string };
 
 export type ConditionalUpdate = {
-  release: Release;
+  release: VersionUpdate;
```

Comment:
@yapei I doubt if this change alone would fix the issue. Did you verify the change?

Reply:
Updated based on comments